nixos/modules/misc/ids.nix

```diff
@@ -486,19 +486,19 @@ in
       mopidy = 130;
       docker = 131;
       gdm = 132;
-      #dhcpcd = 133; # unused
+      dhcpd = 133;
```

Comment:
I guess changing the name here was intentional?

Reply:
The user with the same uid is dhcpd, so I think the c in the comment was a typo.